### ISSUE TITLE
Dashboard comments 28Dec2017

### ISSUE THREAD

**Issue by nita24** [rburghol/om_agman#2]
![image](https://user-images.githubusercontent.com/23525183/34425704-daea51ba-ec71-11e7-9f17-e44ac9ff08d8.png)

Just want to make sure that we have a function to tell users:
1) precipitation amount, 
2) # of nights with high humidity (we will define, but let's do nights with temp more than 65 and RH more than 80%, wet hours all/dark should be kept too.), and 
3) # of risk event from the previous spray date in the dashboard. 
(It is not happening, but it can be because I don't have the previous spray date... In that case, 24 hour information should be good)

**Comment by rburghol** [comment]
1. Done (how the heck did I miss that??)
2. Yes -- are we going to revisit risk modeling this year? I would like to :)
3. Same as #2 - we definitely want to do this, but I am thinking not until we get the basic system ready and user testing under way.

I will take #2 and #3 and separate those into another issue since I think they are to be tackled in 2018 (or 2019 depending on how much changes we need in response to user testing).

Let me know if the precip looks good and I will close this issue.

**Comment by nita24** [comment]
1. Thanks!
2. Yes, at least on the model level, we need to revisit. We need to make sure they are doing what we are aiming. Ground truthing may require more manpower
3. Sounds goo.

Both precip. and leaf wetness look good so far. Looks like there was a frost event. ;)

**Comment by nita24** [comment]
OK, I think I have a good tentative solution for number 2. 
How about having "dark RH" in the same manner as dark temp? 
There is no official model for dark time sporulation for downy, but we know it is a factor.
Right now we simply do not have any risk information to give, but if we know the RH and temp during dark hours, I can come up with the potential risk.

**Comment by rburghol** [comment]
We can.  I am moving this to the thread on enhanced risk models, but we can implement this change in short term since we already include RH in the dark period summary.
